dashboard/client/components/+custom-resources/crd-resource-details.tsx

```diff
@@ -18,6 +18,21 @@ import { KubeObjectMeta } from "../kube-object/kube-object-meta";
 interface Props extends KubeObjectDetailsProps<KubeObject & { status: any }> {
 }
 
+function CrdColumnValue({ value }: { value: any[] | {} | string }) {
+  if (Array.isArray(value)) {
+    return <>{value.map((item, index) => <CrdColumnValue key={index} value={item} />)}</>
```

Comment:
```suggestion
    return value.map((item, index) => <CrdColumnValue key={index} value={item} />)
```

Reply:
The reason for using the fragment is because the component is recursively calling itself and the typings don't like it without. I am sure it could be made to work but this is the simplest solution.